src/Components/HomePage.js

```diff
@@ -93,28 +96,86 @@ const Fitness = props => {
   );
 };
 
+const Theory = props => {
+  return (
+    <View style={styles.mainContainer}>
+      <TouchableHighlight onPress={() => props.navigation.navigate('DictionaryPage')} style={styles.menuItem}>
+        <ImageBackground source={dictionary} style={styles.image}>
+          <View style={styles.wrapper}>
+            <Text style={styles.text}>{I18n.t('homePage.dictionary')}</Text>
+          </View>
+        </ImageBackground>
+      </TouchableHighlight>
+      <TouchableHighlight
+        onPress={() => props.navigation.navigate('EssentialPage', { type: DrillTypes.FRISBEE })}
+        style={styles.menuItem}
+      >
+        <ImageBackground source={essential} style={styles.image}>
+          <View style={styles.wrapper}>
+            <Text style={styles.text}>{I18n.t('homePage.essential')}</Text>
+          </View>
+        </ImageBackground>
+      </TouchableHighlight>
+      <TouchableHighlight onPress={() => props.navigation.navigate('TacticsPage')} style={styles.menuItem}>
+        <ImageBackground source={simulator} style={styles.image}>
+          <View style={styles.wrapper}>
+            <Text style={styles.text}>{I18n.t('homePage.tactics')}</Text>
+          </View>
+        </ImageBackground>
+      </TouchableHighlight>
+    </View>
+  );
+};
+
 export default HomePage = props => {
+  const { navigation } = props;
+
+  const displayEditor = () => {
+    return (
+      <TouchableOpacity
+        style={styles.headerContainer}
+        onPress={() => props.navigation.navigate('AnimationEditorPage')}
+        testID="editorButton"
+      >
+        <MaterialCommunityIcons name="email-outline" color={theme.COLOR_PRIMARY} size={26} />
+      </TouchableOpacity>
+    );
+  };
+
+  useLayoutEffect(() =>
+    navigation.setOptions({
+      headerRight: () => displayEditor(),
+    }),
+  );
   return (
     <Tab.Navigator
       initialRouteName="Frisbee"
-      activeColor={theme.COLOR_PRIMARY}
+      activeColor={theme.COLOR_PRIMARY_LIGHT}
```

Comment:
It changes a lot the UI. The whole application has a light background color

Maybe we should discuss this with vdd

Reply:
Yes, we will. I felle that the tab isn't obvious enough with the light background